worker/src/index.js

```diff
@@ -0,0 +1,46 @@
+const Config = require('../config.json');
+const rp = require('request-promise');
+const rabbit = require('amqplib').connect(Config.RABBIT_URL);
+
+async function consumeTasks() {
+  try {
+    const conn = await rabbit;
+    const channel = await conn.createChannel();
+    await channel.assertQueue('bookmarks');
+
+    channel.consume('bookmarks', (bookmark) => {
+      checkBookmark(JSON.parse(bookmark.content.toString()));
```

Comment:
I think you want `await checkBookmark()`, so that `checkBookmark` completes before you continue